### DISCUSSION TITLE
using collabra for exporting pdf

### DISCUSSION THREAD

**Discussion by jmls** [CollaboraOnline/online#10733] [Q&A] [answered]
I am trying to use soffice to convert a docx to pdf. Unfortunately this docx has altChunks, and libreoffice does not support that. 

I saw that Collabora Office 6.2-26  supports "altChunk (embedded DOCX in DOCX)", and was wondering if it would be possible for collabra to be run using docker where it's sole purpose is to convert a docx to pdf , similar to 

soffice --headless --convert-to pdf --infilter="MS Word 2007 XML" my.docx

**Comment by vmiklos** [comment, marked as answer]
I guess there are multiple types of altChunks, support for one specific type was added, but your docx may be a different one. Certainly the code that was added to Collabora Office 6.2 (years ago) is still there with a recent CODE or COOL container.

Probably best to file an issue with a reproducer document.
